src/main/scala/diplomaticobjectmodel/logicaltree/LogicalTrees.scala

```diff
@@ -46,10 +57,41 @@ class DebugLogicalTreeNode(device: SimpleDevice, f: => OMRegisterMap, debugModul
             version = "0.13"
           )
         ),
+        interfaceType = OMDebug.getOMDebugInterfaceType(p),
+        nSupportedHarts = nComponents,
         nAbstractDataWords = cfg.nAbstractDataWords,
         nProgramBufferWords = cfg.nProgramBufferWords,
-        interfaceType = OMDebug.getDebugInterfaceType(exportDebugJTAG, exportDebugCJTAG, exportDebugDMI),
-      )
+        nDMIAddressSizeBits = cfg.nDMIAddrSize,
+        hasSystemBusAccess = cfg.hasBusMaster,
+        supportsQuickAccess = cfg.supportQuickAccess,
+        supportsHartArray = cfg.supportHartArray,
+        hasImplicitEbreak = cfg.hasImplicitEbreak,
+        sbcsSBAVersion = 1,
+        sbaAddressSizeBits = cfg.maxSupportedSBAccess,
+        hasSBAccess8 = cfg.maxSupportedSBAccess >= 8,
+        hasSBAccess16 = cfg.maxSupportedSBAccess >= 16,
+        hasSBAccess32 = cfg.maxSupportedSBAccess >= 32,
+        hasSBAccess64 = cfg.maxSupportedSBAccess >= 64,
+        hasSBAccess128 = cfg.maxSupportedSBAccess == 128,
+        hartSeltoHartIDMapping = Nil, // HartSel goes from 0->N but HartID is not contiguious or increasing
+        authenticationType = NONE,
+        nHartsellenBits = p(MaxHartIdBits), // Number of actually implemented bits of Hartsel
+        hasHartInfo = true,
+        hasAbstractauto = true,
+        cfgStrPtrValid = false,
+        nHaltSummaryRegisters = 2,
+        nHaltGroups = cfg.nHaltGroups,
+        nExtTriggers = cfg.nExtTriggers,
+        hasResetHaltReq = true,
```

Comment:
Hi Derek,
A couple of these fixed-value items do not have the right values for the current DM implementation.  They should be:
hasResetHaltReq = false
hasAbstractPostExec = true

Reply:
Hi Ernie,

Please make a PR to fix the values.

Reply:
https://github.com/freechipsproject/rocket-chip/pull/1938